docs/content/guides/operator/updates.mdx

```diff
@@ -3,13 +3,32 @@ title: Updating a Full Node
 description: Update your Sui Full node to the latest version to remain in sync with the network.
 ---
 
-## Release process
+## Sui release process
+
+Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through [official channels](#communication).
+
+- `devnet`: Deployed every week on Mondays.
+- `testnet`: Deployed every week on Tuesdays.
+- `mainnet`: Deployed every two weeks on Wednesdays.
+
+:::info
+ For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
+ :::
```

Comment:
@ronny-mysten this does not render correctly
<img width="744" alt="Screenshot 2024-10-15 at 3 22 10 PM" src="https://github.com/user-attachments/assets/aba43bc8-b1b9-42ce-b1bc-85aa78c470fc">